server/controllers/gift.controller.js

```diff
@@ -7,20 +7,19 @@ const Guest = DB.Guest;
 const { messages } = require('./../helper');
 
 const isEventOwner = (eventId, userId, gift) => {
-  return Event.findOne({where: {id: +eventId, $and: {owner: userId}}})
-  .then(event => !!event && !!gift && (+gift.dataValues.event_id === +eventId))
+  return Event.findOne({where: {id: eventId, $and: {owner: userId}}})
```

Comment:
In a concise body, only an expression is needed, and an implicit return is attached. In a block body, you must use an explicit return statement.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions